v1/lib/core/DocsLayout.js

```diff
@@ -76,6 +76,34 @@ class DocsLayout extends React.Component {
         language={metadata.language}
         version={metadata.version}
         metadata={metadata}>
+        {
+          <script
+            dangerouslySetInnerHTML={{
+              __html: `
+                function openConfig(num) {
+                  var i, tabContent, tabLinks;
+                  tabContent = document.getElementsByClassName("tabcontent");
+                  for (i = 0; i < tabContent.length; i++) {
+                      tabContent[i].style.display = "none";
+                  }
+                  tabLinks = document.getElementsByClassName("tablinks");
+                  for (i = 0; i < tabLinks.length; i++) {
+                      tabLinks[i].className = tabLinks[i].className.replace(" active", "");
```

Comment:
Let's be more specific and use `$` regex
```js
replace(/ active$/, "")
```

This is to avoid replacing a very special edge case `nav activeItem active` to `navItem active` in which it should be `nav activeItem`